src/main/kotlin/io/github/compose4gtk/gtk/components/CheckButton.kt

```diff
@@ -1,16 +1,27 @@
 package io.github.compose4gtk.gtk.components
 
 import androidx.compose.runtime.*
-import io.github.jwharm.javagi.gobject.SignalConnection
 import io.github.compose4gtk.GtkApplier
 import io.github.compose4gtk.SingleChildComposeNode
 import io.github.compose4gtk.modifier.Modifier
+import io.github.jwharm.javagi.gobject.SignalConnection
 import org.gnome.gtk.CheckButton
 
 private class GtkCheckButton(gObject: CheckButton) : SingleChildComposeNode<CheckButton>(gObject, { child = it }) {
     var toggled: SignalConnection<CheckButton.ToggledCallback>? = null
 }
 
+private val LocalCheckButtonGroupLeader = compositionLocalOf<MutableState<CheckButton?>?> { null }
```

Comment:
Why is the mutability required?
I'd suggest just:
```suggestion
private val LocalCheckButtonGroupLeader = compositionLocalOf<CheckButton?> { null }
```

Reply:
Because I change its value here:

``` kotlin
if (groupLeaderState != null) {
    if (groupLeaderState.value != null) {
        cb.setGroup(groupLeaderState.value)
    } else {
           groupLeaderState.value = cb
        cb.setGroup(cb)
    }
}
```